tla/validate-model.sh

```diff
@@ -22,16 +22,23 @@ function validate {
     local config=${2}
     local tooldir=${3}
     local statedir=${4}
+    local sim=${5:-false}
 
     set -o pipefail
-    java -XX:+UseParallelGC -cp ${tooldir}/tla2tools.jar:${tooldir}/CommunityModules-deps.jar tlc2.TLC -config "${config}" "${spec}" -lncheck final -metadir "${statedir}" -fpmem 0.9
+    if [ "$sim" ]; then
```

Comment:
The simulate flag is added to the else part. It should be here no?

Reply:
Also, it should be "$sim" = true, otherwise it is always true.

Reply:
You're correct. simulate flag shall be added when $sim is true. Fix it in the new commit.